stdlib/public/Darwin/SpriteKit/SpriteKit.swift

```diff
@@ -62,7 +62,7 @@ extension SKWarpGeometryGrid {
   /// - Parameter rows: the number of rows to initialize the SKWarpGeometryGrid with
   /// - Parameter sourcePositions: the source positions for the SKWarpGeometryGrid to warp from
   /// - Parameter destinationPositions: the destination positions for SKWarpGeometryGrid to warp to
-  public convenience init(columns: Int, rows: Int, sourcePositions: [simd.float2] = [float2](), destinationPositions: [simd.float2] = [float2]()) {
+  public convenience init(columns: Int, rows: Int, sourcePositions: [SIMD2<Float>] = [SIMD2<Float>](), destinationPositions: [SIMD2<Float>] = [SIMD2<Float>]()) {
```

Comment:
Can the default arguments use empty array literals?

```suggestion
  public convenience init(columns: Int, rows: Int, sourcePositions: [SIMD2<Float>] = [], destinationPositions: [SIMD2<Float>] = []) {
```

Reply:
They could, but I'd prefer to keep this PR restricted to *just* updating the type names.